artifactory-enterprise/nested/azureDBDeploy.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "db_user": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "artifactory"
```

Comment:
remove all of the defaultValues here since they must be passed through

Reply:
done